packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue

```diff
@@ -348,6 +348,7 @@ footer {
 
 :deep(.p-chip .p-chip-text) {
 	font-size: var(--font-body-small);
+	margin: 2px;
```

Comment:
```suggestion
	margin: var(--gap-0-5);
```

See our gap variables: https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L986C2-L986C11